rootfs/etc/nginx/lua/balancer/ewma.lua

```diff
@@ -0,0 +1,128 @@
+-- Original Authors: Shiv Nagarajan & Scott Francis
+-- Accessed: March 12, 2018
+-- Inspiration drawn from:
+-- https://github.com/twitter/finagle/blob/1bc837c4feafc0096e43c0e98516a8e1c50c4421/finagle-core/src/main/scala/com/twitter/finagle/loadbalancer/PeakEwma.scala
```

Comment:
@shivnagarajan @csfrancis please let us know if you have any comment regarding to this.

Reply:
Sorry I am late to look at this. But that reference looks 💯